Tests/TuistHasherTests/GraphContentHasherTests.swift

```diff
@@ -100,6 +99,118 @@ final class GraphContentHasherTests: TuistUnitTestCase {
         .sorted(by: { $0.target.name < $1.target.name })
 
         // Then
-        XCTAssertEqual(hashedTargets, expectedCachableTargets)
+        #expect(hashedTargets == expectedCachableTargets)
+    }
+
+    @Test
+    func test_contentHashes_with_lock_file_in_xcode_project() async throws {
+        try await fileSystem.runInTemporaryDirectory(prefix: "GraphContentHasherTests") { path in
```

Comment:
I'd codify in the variable name its semantics:

```suggestion
        try await fileSystem.runInTemporaryDirectory(prefix: "GraphContentHasherTests") { temporaryDirectory in
```